thirdparty/patches/boost-exception-no_warn_typeid_evaluated.patch

```diff
@@ -1,10 +0,0 @@
-diff --git boost/exception/info.hpp boost/exception/info.hpp
```

Comment:
Why we remove this file? Is it out of date in the new version?

Reply:
Yes, there's no such assertion anymore in [boost 1.79.0](https://github.com/boostorg/exception/blob/boost-1.79.0/include/boost/exception/info.hpp),  and I've search the usage of `BOOST_EXCEPTION_DYNAMIC_TYPEID` to ensure that it's not just been moved to other places.